### PR TITLE
Update 5TB section

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/change-user-storage.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ When a user has filled their 5 TB of OneDrive storage to at least 90% capacity,
 
 For any user that reaches at least 90% capacity of their 25 TB of OneDrive storage, additional cloud storage will be provided as 25 TB SharePoint team sites to individual users. 
 
-Admins can [open a case with Microsoft technical support](/microsoft-365/admin/contact-support-for-business-products) to request storage beyond 5 TB.
+Admins can check for OneDrive eligibility beyond 5TB via [Check OneDrive site eligibility for increased storage](https://docs.microsoft.com/en-us/sharepoint/troubleshoot/storage/check-storage-increase-eligibility).
```
Comment (JohanFreelancer9) [inline OneDrive/change-user-storage.md:63]:
```suggestion
Admins can check for OneDrive eligibility beyond 5 TB via [Check OneDrive site eligibility for increased storage](/sharepoint/troubleshoot/storage/check-storage-increase-eligibility). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the suggested changes. Many thanks.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**